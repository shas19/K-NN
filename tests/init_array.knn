let t = initArray(4) in 
let t[2] = 4.0 in
let b = 5.0 in 
t[2]