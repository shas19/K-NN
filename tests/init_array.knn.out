<T>
  <k>
    0.40000000000000000e1 ~> .
  </k>
  <env>
    t |-> 0
  </env>
  <store>
    0 |-> array ( 1 , 4 )
    1 |-> 0.00000000000000000e0
    2 |-> 0.00000000000000000e0
    3 |-> 0.40000000000000000e1
    4 |-> 0.00000000000000000e0
  </store>
  <nextLoc>
    5
  </nextLoc>
  <shape>
    t |-> 4
  </shape>
</T>
